src/patternfly/components/Label/label.scss

```diff
@@ -145,16 +146,18 @@
   --pf-c-label__icon--Color: var(--pf-global--Color--100);
   --pf-c-label__icon--MarginRight: var(--pf-global--spacer--xs);
 
-  // Close button
-  --pf-c-label__c-button--FontSize: var(--pf-global--FontSize--xs);
-  --pf-c-label__c-button--MarginTop: calc(var(--pf-global--spacer--form-element) * -1);
-  --pf-c-label__c-button--MarginRight: calc(var(--pf-global--spacer--form-element) * -1);
-  --pf-c-label__c-button--MarginBottom: calc(var(--pf-global--spacer--form-element) * -1);
-  --pf-c-label__c-button--MarginLeft: var(--pf-global--spacer--xs);
-  --pf-c-label__c-button--PaddingTop: var(--pf-global--spacer--xs);
-  --pf-c-label__c-button--PaddingRight: var(--pf-global--spacer--sm);
-  --pf-c-label__c-button--PaddingBottom: var(--pf-global--spacer--xs);
-  --pf-c-label__c-button--PaddingLeft: var(--pf-global--spacer--sm);
+  // Actions
+  --pf-c-label__actions--FontSize: var(--pf-global--FontSize--xs);
+  --pf-c-label-actions--MarginRight: calc(var(--pf-global--spacer--form-element) * -1);
+
+  // Actions button (close)
+  --pf-c-label-actions--c-button--FontSize: var(--pf-global--FontSize--xs);
+  --pf-c-label-actions--c-button--MarginTop: calc(var(--pf-global--spacer--form-element) * -1);
+  --pf-c-label-actions--c-button--MarginBottom: calc(var(--pf-global--spacer--form-element) * -1);
+  --pf-c-label-actions--c-button--PaddingTop: var(--pf-global--spacer--xs);
+  --pf-c-label-actions--c-button--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-label-actions--c-button--PaddingBottom: var(--pf-global--spacer--xs);
+  --pf-c-label-actions--c-button--PaddingLeft: var(--pf-global--spacer--sm);
```

Comment:
While the margin-right looks ok visually, it's kind of an arbitrary choice in spacer as far as I can tell. It looks like the margins that reference the form-element spacer may just be out of date?

If we want to keep that same visual spacing, I think this formula for that negative margin works better and would scale well? Seems to scale as long as the action's right padding is greater than the right padding of the label.

```suggestion
  --pf-c-label__actions--MarginRight: calc(var(--pf-c-label-actions--c-button--PaddingRight) * -1 + var(--pf-c-label--PaddingRight) / 2);

  // Actions button (close)
  --pf-c-label__actions--c-button--FontSize: var(--pf-global--FontSize--xs);
  --pf-c-label__actions--c-button--MarginTop: calc(var(--pf-c-label__actions--c-button--PaddingTop) * -1);
  --pf-c-label__actions--c-button--MarginBottom: calc(var(--pf-c-label__actions--c-button--PaddingBottom) * -1);
  --pf-c-label__actions--c-button--PaddingTop: var(--pf-global--spacer--xs);
  --pf-c-label__actions--c-button--PaddingRight: var(--pf-global--spacer--sm);
  --pf-c-label__actions--c-button--PaddingBottom: var(--pf-global--spacer--xs);
  --pf-c-label__actions--c-button--PaddingLeft: var(--pf-global--spacer--sm);
```

Just a side note on that margin, ideally I think it could be setup like the label icon instead where the icon touches the start of the label's padding, regardless the size or padding or whatever. I think to do that, the actions margin-right would just be `calc(var(--pf-c-label-actions--c-button--PaddingRight) * -1)`, but that ends up looking a little tight on the spacing, so we could add `.fa-fw` to the close icons - that class also exists on the label icon. But that would need to be tested in react, too, to make sure it displays correctly.